cli/progress/multiprogress.go

```diff
@@ -174,6 +175,7 @@ func (m modelMultiProgress) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 			m.Progress[msg.Index].lock.Unlock()
 		}
 
+		overallPercent := 0.0
 		if _, ok := m.Progress[overallProgressIndexName]; ok {
 			m.Progress[msg.Index].lock.Lock()
```

Comment:
why is the lock on the index and not the overallProgressIndexName

Reply:
Good catch. Thanks. That was a Copy&Paste error.